internal/colorize/cropped.go

```diff
@@ -55,7 +67,7 @@ func GetCroppedText(text string, maxLen int, includeLineEnds bool) CroppedLines
 				}
 				// Extract the word from the current line if it doesn't start the line.
 				if i > 0 && i < len(entry.Line)-1 {
-					wrapped = entry.Line[i:]
+					wrapped = indent + entry.Line[i:]
```

Comment:
Not only will this work for change summaries, but it will work for ANY indented output -- subsequent lines will be indented.